source/_posts/2024-05-22-companion-app-for-ios-20245-assist.markdown

```diff
@@ -0,0 +1,61 @@
+
+### “What if I don’t have an iPhone 15 pro to use the action button?”
+
+It would be ideal if we could replace Siri with Assist in the power button long press, but since Apple doesn’t allow that. The second best option becomes the action button. For iPhones without it, there is a third option, the accessibility feature called “back tap”. Follow [this guide on the Apple page](https://support.apple.com/en-gb/guide/shortcuts/apd897693606/ios) to learn how to configure it.
+
+### Widgets
+
+If back tap doesn’t work for you, Assist can be activated via our widgets from the home screen or with our very first iOS lock screen widget, which looks great:
+
+![Assist Widgets](/images/blog/2024-05-ios-assist/widgets.png)
+
+### Accessibility feature alternative
+
+There is “one more thing” that you can use to launch Assist, iOS has an accessibility feature called “Voice Control” which allows a user to basically navigate an iPhone completely using voice commands. On top of that they allow you to add “custom commands” so… Yes! You can just add “Ok Nabu”, then use the option “Run shortcut” and have it trigger “Assist In App”.
+
+A couple things to take into consideration. I have tried this for the last weekend and although it’s pretty cool, it’s not as fast as calling Siri.  You will also have full voice control enabled, so  you trigger other commands every once in a while.
+
+<lite-youtube videoid="ovFqaIiefuo" videotitle="Assist in iOS using accessibility voice control"></lite-youtube>
+
+## Other highlights
+
+- We have introduced the ability to run your Action widgets in the background, so you can quickly trigger them instead of having to wait for the app to launch to see the result.
+
+- This version introduced a simple way to create an automation for your iOS Action: simply tap on the “create automation” button.
+
+- We started improving our widgets' performance and capabilities. I know you all have great ideas, and I am currently working on the foundation to allow those to happen, but please continue sending feature requests to our Home Assistant Community.
+
+- When we first released CarPlay integration, we missed out on displaying all areas due to a limitation in the number of items allowed on a list. Since 2024.2, we have worked around this by using the first and last rows of the areas list as pagination arrows, so you can now see all your rooms.
+
+- The Actions UI in CarPlay was aligned to display the same information as in Apple Watch, previously we were displaying the “Identifier” and ”Text”, now we display just the “Text” so we keep consistency between platforms.
+
+- We also added a button to reload your widgets, since sometimes they may become blank for a few users after an update. For actions, we added a button to manually update server-created Actions so you don't need to reopen the app for that (but remember to restart HA after adding or removing Actions from yaml).
```

Comment:
```suggestion
- We also added a button to reload your widgets, since sometimes they may become blank for a few users after an update. For actions, we added a button to update server-created Actions manually, so you don't need to reopen the app for that (but remember to restart HA after adding or removing Actions from YAML).
```